openpype/hosts/flame/plugins/publish/collect_timeline_instances.py

```diff
@@ -126,6 +148,94 @@ def process(self, context):
                 if marker_data.get("reviewTrack") is not None:
                     instance.data["reviewAudio"] = True
 
+    def _get_comment_attributes(self, segment):
+        comment = segment.comment.get_value()
+
+        # try to find attributes
+        attributes = {
+            "xml_overrides": {
+                "pixelRatio": 1.00}
+        }
+        # search for `:`
+        for split in self._split_comments(comment):
+            # make sure we ignore if not `:` in key
+            if ":" not in split:
+                continue
+
+            self._get_xml_preset_attrs(
+                attributes, split)
+
+        # add xml overides resolution to instance data
+        xml_overrides = attributes["xml_overrides"]
+        if xml_overrides.get("width"):
+            attributes.update({
+                "resolutionWidth": xml_overrides["width"],
+                "resolutionHeight": xml_overrides["height"],
+                "pixelAspect": xml_overrides["pixelRatio"]
+            })
+
+        return attributes
+
+    def _get_xml_preset_attrs(self, attributes, split):
+
+        # split to key and value
+        key, value = split.split(":")
+
+        for a_name, a_type in self.xml_preset_attrs_from_comments.items():
+            # exclude all not related attributes
+            if a_name.lower() not in key.lower():
+                continue
+
+            # get pattern defined by type
+            pattern = TXT_PATERN
+            if "number" in a_type or "float" in a_type:
```

Comment:
Don't use for string comparison `"number" in "not-number"` if you're expecting specific value.
```suggestion
            if a_type in ("number" , "float"):
```